File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/userprofile/VehicleInsurancePolicyUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests.userprofile;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.sql.Date;

@Data
public class VehicleInsurancePolicyUpdateRequest {

    @NotNull
    @NotEmpty
    private final String updatedVehicleInsuranceCompany;

    @NotNull
    @NotEmpty
    private final String updatedVehicleInsurancePolicyNumber;

    @NotNull
    private final Date updatedVehicleInsurancePolicyExpirationDate;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/VehicleInsurancePolicyUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

import java.sql.Date

data class VehicleInsurancePolicyUpdateRequest(
    val updatedVehicleInsuranceCompany: String,
    val updatedVehicleInsurancePolicyNumber: String,
    val updatedVehicleInsurancePolicyExpirationDate: Date
)
